### PR TITLE
Adds ability to change canvas' background color and pen color through props

### PR DESCRIPTION
**Pull request by holtat**
Adds props to SignaturePad component for setting the backgroundColor and penColor.

Example:

```
<SignaturePad
  backgroundColor="#ba5a55"
  penColor="#0054e3"
  onChange={this.onChange}
/>
```

Renders:
screen shot 2016-06-10 at 9 36 18 am

The default fall back colors are still black and white:

```
<SignaturePad
  onChange={this.onChange}
/>
```

Renders:
screen shot 2016-06-10 at 9 44 51 am

Unfortunately, you cannot set the backgroundColor through the style prop because we cannot guarantee that it will be a string. The style prop, if created as a Stylesheet, will be passed as number and we are unable to see the `backgroundColor` value.

### CODE REVIEW

**Comment by kevinstumpf** [comment]
@holtat Thank you for the contribution!

> Unfortunately, you cannot set the backgroundColor through the style prop because we cannot guarantee that it will be a string. The style prop, if created as a Stylesheet, will be passed as number and we are unable to see the backgroundColor value.

Take a look at StyleSheet.flatten:
https://github.com/facebook/react-native/blob/e79f5d7e7a129f30a8817c0676c98990fbe01686/Libraries/StyleSheet/StyleSheet.js#L119

I believe that helper allows you to retrieve the backgroundColor from the style prop. That might be a better approach, what do you think?

Best,
Kevin

**Comment by holtat** [comment]
@kevinstumpf 

Thanks for pointing me there. I didn't know that existed.

I've refactored to use the backgroundColor from the style props. I tried to use the color key on the style props, but it is not a valid View style key. I chose to stick with the penColor prop.

Let me know what you think.

**Comment by kevinstumpf** [comment]
> I've refactored to use the backgroundColor from the style props. I tried to use the color key on the style props, but it is not a valid View style key

You can fix that by either defining your custom propTypes... or, for simplicity's sake, changing the "style prop type" from View.propTypes.style to Text.propTypes.style, which exposes a "color" field. What do you think?

**Comment by holtat** [comment]
I did know about that and considered changing it to `Text.propTypes.style`, but the SignaturePad corresponds more to a View than a Text. I think it makes the most sense keeping it as a separate prop, but am willing to write it the way that you would prefer.

**Comment by kevinstumpf** [comment]
Good point. What do you think about introducing "SignaturePadPropTypes", which is derived from View PropTypes and simply adds an additional "color" field?

You can implement that similarly to how the Text component's PropTypes are derived from View's PropTypes:
https://github.com/facebook/react-native/blob/0.27-stable/Libraries/Text/Text.js#L98
and 
https://github.com/facebook/react-native/blob/02578df4f731b411b7b0be5f5dd83bd7233d2800/Libraries/Text/TextStylePropTypes.js

**Comment by holtat** [comment]
Interesting idea.

I don't think that we can import 'StyleSheetPropType' since it isn't a RN export.

https://github.com/facebook/react-native/blob/master/Libraries/react-native/react-native.js

If you have used this before, I'd love to learn how to import it. 

`import { StyleSheetPropType } from 'react-native'` gave me an undefined StyleSheetPropType.

I was trying to implement your suggestion similar to the [picker](https://github.com/facebook/react-native/blob/235b16d93287061a09c4624e612b5dc4f960ce47/Libraries/Components/Picker/Picker.js#L29)

**Comment by kevinstumpf** [comment]
Arg... it's not exported: https://github.com/facebook/react-native/blob/803cb61346bcdf85d68df5cdfc1a392f78e6cac3/Libraries/react-native/react-native.js

What a bummer... after a quick skim of the codebase it doesn't look like there's an easy way to access it without modifying react-native's exports.

Ok let's leave it as is then :)

File: index.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ class SignaturePad extends Component {
     onError: PropTypes.func,
     style: View.propTypes.style,
     penColor: PropTypes.string,
+    exportBackgroundColor: PropTypes.string,
```
Comment (kevinstumpf) [inline index.js]: Is this used anywhere?

Comment (holtat) [inline index.js]: It accidentally snuck in there. Already removed it (https://github.com/DispatcherInc/react-native-signature-pad/pull/5/commits/e0553ea77811d7d1787c8e55ba6fd28e5ea03496)

Sorry about that.

**Comment by kevinstumpf** [comment]
Thanks again! Published a release: https://github.com/DispatcherInc/react-native-signature-pad/releases/tag/0.0.7

**Pull request merged**